### PR TITLE
Update in README

### PR DESCRIPTION
**Pull request by amoreno-sonatype**
Updating some instructions and providing examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,25 +7,38 @@ to an OpenShift cluster.
...
    1. `./scripts/install.sh`
+   2. By executing `kubectl get pods` you should see a pod running in Openshift:
+
+		`example-nexusiq-iqserver-{id}`
 8. Expose the new IQ Server outside the cluster: 
-   1. Create a Route in OpenShift UI to the new service, port 8070.
-9. Visit the new URL shown on the Route page in OpenShift UI.
+   1. Create a Route in OpenShift UI to the new service, using:
+      
+      **Port:** 8070 → 8070.
```
Comment (jflinchbaugh) [inline README.md:35]: I'd shy away from that arrow character that's not on the keyboard, and favor instead `->`.

File: README.md [hunk 2]
```diff
@@ -7,25 +7,38 @@ to an OpenShift cluster.
...
+
+      **Service:** example-nexusiq-iqserver-{id}
+9. Visit the new URL shown on the Route page in OpenShift UI. And verfify everything on the IQ Server side is working as expected.
 10. Default credentials are `admin`/`admin123`.
+
+(*) You can get the tag of Nexus IQ Server Certified Image from [https://catalog.redhat.com/](https://catalog.redhat.com/) → Software → Container images → Find Nexus IQ Server. And the image id in the **Digest** section of this page.
```
Comment (jflinchbaugh) [inline README.md:41]: you can reliably deep link to the "Get This Image" page for the image: https://catalog.redhat.com/software/containers/sonatype/nexus-iq-server/5e5d8063ac3db90370816c66?container-tabs=gti

**Review by jflinchbaugh — APPROVED** [review]
+1

File: README.md (hunk 2 above)
Comment (amoreno-sonatype) [reply README.md]: Thanks John. Improved in 4aace4c

**Pull request merged**